builder/ebs/builder_acc_test.go

```diff
@@ -7,58 +7,152 @@ package ebs
 import (
 	"fmt"
 	"os"
+	"os/exec"
 	"testing"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/hashicorp/packer-plugin-amazon/builder/common"
-	builderT "github.com/hashicorp/packer-plugin-sdk/acctest"
-	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+	"github.com/hashicorp/packer-plugin-sdk/acctest"
 )
 
-func TestBuilderAcc_basic(t *testing.T) {
-	builderT.Test(t, builderT.TestCase{
-		PreCheck: func() { testAccPreCheck(t) },
-		Builder:  &Builder{},
+func TestAccBuilderAcc_basic(t *testing.T) {
```

Comment:
nit: this name kind of stutters. Did this and the other TestBuilder named tests get auto-changed as part of a search and replace?

Reply:
Oh, you're right, I just replaced it and didn't notice it was this way. Thanks for catching this.